addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/ValueGenerator.java

```diff
@@ -41,13 +42,10 @@ public String getValue(String name, String type, String defaultValue) {
         if (coreValGen == null) {
             if (LOG.isDebugEnabled()) {
                 LOG.debug(
-                        "Name : "
-                                + name
-                                + " Type : "
-                                + type
-                                + " Default : "
-                                + defaultValue
-                                + " Returning default value");
+                        "Name : {}\nType : {}\nDefault : {}\nReturning default value",
```

Comment:
These changes should be noted in the commit message, but not sure it's needed. The above guard can be removed (same in other places).